package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@adobe/httptransfer",
   "version": "2.1.1",
   "main": "index.js",
+  "browser": "./dist/es2015/exports.js",
```

Comment:
Should this be `./es2015/exports.js`?

Also, I know I named the file, but the more I thought about it the more I wondered if it might be more convenient to change `exports.js` to `index.js`? That way consumers could do `require('httptransfer/es2015')` instead of `require('httptransfer/es2015/exports)`. WDYT?

Reply:
Sounds good